materialize-postgres/.snapshots/TestValidateAndApply

```diff
@@ -193,14 +193,14 @@ Big Schema Materialized Resource Schema With No Fields Required:
 
 Big Schema Changed Types With Table Replacement Constraints:
 {"Field":"_meta/flow_truncated","Type":4,"TypeString":"FIELD_OPTIONAL","Reason":"Metadata fields fields are able to be materialized"}
-{"Field":"arrayField","Type":4,"TypeString":"FIELD_OPTIONAL","Reason":"This field is able to be materialized"}
+{"Field":"arrayField","Type":4,"TypeString":"FIELD_OPTIONAL","Reason":"Object fields may be materialized"}
```

Comment:
Is this normal that some of these snapshots have `arrayField` as optional?

Reply:
Yeah, similar to the explanation from https://github.com/estuary/connectors/pull/1209#discussion_r1466616604, the field called `arrayField` isn't really an array at this point in the test lifecycle, but has had its collection schema changed to have it be an Object and incrementing the backfill counter has it be validated as a few field for the materialization.